website/static/custom.css

```diff
@@ -68,9 +68,27 @@
     max-width: 200px;
 }
 
+ol {
+  /*border-top: 1px solid #D80029;
+  border-bottom: 1px solid #F82049;
+  font-style: italic;*/
+  background-color: rgba(216, 0, 41, 0.05);
+  border-left: 8px solid rgba(216, 0, 41, 0.2);
+  color: #555;
```

Comment:
I'm not sure about this - it does make some of the text hard to read, especially with blue on red and red links on red, just thinking about accessibility and usability. Also red is often associated with warnings.

Reply:
Based on previous comments on the print manual, I think the block of 1-2-3 instructions should be visually tied together somehow to set it apart from the other text, but you may be right about the background. The print manuals just have a vertical line to the left - that would probably be OK. I used red as the Avolites style colour ... but it doesn't have to be.